utils.py

```
@@ -108,6 +108,11 @@ def cif2cmap(pdb, chain, seq, pdir):
 
 def write_annot_npz(prot, prot2seq, struct_dir, is_csm=True):
     print("Debug prot:", prot)
+
+    if len(prot.split('_')) > 1:
+        is_csm = True
+    else:
+        is_csm = False
```

Ruff

Use ternary operator `is_csm = True if len(prot.split("_")) > 1 else False` instead of `if`-`else`-block

Replace `if`-`else`-block with `is_csm = True if len(prot.split("_")) > 1 else False`

(SIM108)